tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+	"compilerOptions": {
```

Comment:
We _may_ need `"resolveJsonModule": true,` in there too, based on the TypeScript I've been writing in another project.